src/assets/Group.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="23.2" height="21.6" viewBox="0 0 29 27" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Group.svg isn't a very descriptive name, could you change it to "arrow.svg" or something similar?

Reply:
I changed it to HeroArrow.svg